pkg/collector/corechecks/cluster/orchestrator/processors/k8s/cluster.go

```diff
@@ -149,6 +150,7 @@ func (p *ClusterProcessor) Process(ctx processors.ProcessorContext, list interfa
 			GroupId:     pctx.MsgGroupID,
 			Cluster:     clusterModel,
 			Tags:        pctx.Cfg.ExtraTags,
+			AgentInfo:   common.K8sAgentInfo(pctx.InstallInfo),
```

Comment:
I still fill like it is not the right place to collect agent information. 

What happen if the check run in a cluster-agent? do we still this information?

Could you explain the need of collecting this information 🙇 
To give some context we plan to collect cluster-agent configuration in a near futur (Q4) and expose the info in REDAPL. The cluster-agent config can contain the install method. Can it cover the same need? does the timeline can work with yours?

Reply:
> What happen if the check run in a cluster-agent? do we still this information?

Yes this information is collected in the cluster-agent


> To give some context we plan to collect cluster-agent configuration in a near futur (Q4) and expose the info in REDAPL. The cluster-agent config can contain the install method. Can it cover the same need? does the timeline can work with yours?

In this case we will probably use this information from REDAPL thank you